atomicfu-gradle-plugin/build.gradle

```
@@ -20,15 +20,15 @@ dependencies {
 
     compileOnly gradleApi()
     compileOnly 'org.jetbrains.kotlin:kotlin-stdlib'
-    compileOnly "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+    compileOnly libs.kotlin.gradlePlugin
     // Atomicfu compiler plugin dependency will be loaded to kotlinCompilerPluginClasspath
     // Atomicfu plugin will only be applied if the flag is set kotlinx.atomicfu.enableJsIrTransformation=true
-    compileOnly "org.jetbrains.kotlin:atomicfu:$kotlin_version"
+    compileOnly "org.jetbrains.kotlin:atomicfu:${libs.versions.kotlin}"
```

I'm not familiar with the Groovy DSL, will this get the actual value, or instead call `Provide<String>.toString()`, which returns something like `"Provider(...)"` instead of `1.9.23`?

In Kotlin you need to `.get()` the value, e.g.:

```suggestion
    compileOnly "org.jetbrains.kotlin:atomicfu:${libs.versions.kotlin.get()}"
```

---

fixed